hs_core/serialization.py

```diff
@@ -171,6 +172,8 @@ def create_resource_from_bag(bag_content_path, preserve_uuid=True):
 
     try:
         rm.write_metadata_to_resource(resource)
+        # Force bag files to be re-written
+        create_bag_files(resource)
```

Comment:
@selimnairb Should this create_bag_files() be called at the end of write_metadata_to_resource() function rather than in create_resource_from_bag()? For example, in my bag_ingest.py migration script, I have to call write_metadata_to_resource() for all dependent resources directly whose metadata did not get created successfully in the first round of loop of calling create_resource_from_bag().